docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -986,10 +986,92 @@ class DetailResource {
 
 In the development mode, all files located in `src/main/resources/templates` are watched for changes and modifications are immediately visible.
 
-=== Configuration Reference
+=== Type-safe Message Bundles
+
+==== Basic Concepts
+
+The basic idea is that every message is potentially a very simple template.
+In order to prevent type-safe errors a message is defined as an annotated method of a *message bundle interface*.
```

Comment:
"prevent type-safe errors" sounds weird :-) You probably meant just "prevent errors".

Reply:
Well, I meant "prevent type errors". Thanks for spotting this.